src/protocol/sort/apply_sort/shuffle.rs

```diff
@@ -81,6 +82,25 @@ where
     try_join_all(reshares).await
 }
 
+#[async_trait]
+impl<F: Field> Resharable<F> for Vec<MaliciousReplicated<F>> {
```

Comment:
It feels like we shouldn't have to special case this. It feels like we should just have one implementation for ANY vector of Reshareables. Maybe we can add that later.

Reply:
Agree on this. Even before that, is it possible to share Resharable for MaliciousReplicated and Replicated? I tried but couldnt get it to working @akoshelev

Reply:
Thanks to @taikiy this works now!!